messages/0.1.5.txt

```diff
@@ -0,0 +1,16 @@
+Welcome to the TypeScript plugin for Sublime Text!
+
+Details about this package and its feature can be found on its GitHub
+repository at https://github.com/Microsoft/TypeScript-Sublime-Plugin.
+
+v0.1.5 updates:
+	- Code base refactored for easier future maintenance
+	- Fixed several issues of the syntax definition file
+	- Fixed performance issues caused by quickinfo requests on large files
+	- Fixed the bug that "}" and ";" won't show up when the formatOnKey command fails
```

Comment:
Agree with Dan.  Please try and word the readme in a manner end users will understand, not just plug-in developers.